Edits to PGD PR5477 (#5518)
EnterpriseDB/docs: product_docs/docs/pgd/5/sequences.mdx

unique ids generated by such sequences cause conflict and data loss by
means of discarded `INSERT` actions in multi-master replication.

!!! Note Permissions required

> PGD global sequences are on by default. This means that, to use sequences, 

Reviewer: This note is making the point that for ANY user to use sequences they must have the appropriate role. Sequences are embedded and other users using the table are required to have the same role privilege - hence for any user rather than the user.

unique ids generated by such sequences cause conflict and data loss by
means of discarded `INSERT` actions in multi-master replication.

!!! Note Permissions required
PGD global sequences are on by default. This means that any user who wants to use sequences, 

> must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) 

Author: @djw-m Does this work? The original was hard to parse because of the two uses of the word "user" in the sentence.

unique ids generated by such sequences cause conflict and data loss by
means of discarded `INSERT` actions in multi-master replication.

!!! Note Permissions required

> PGD global sequences are on by default. This means that, to use sequences, 

Author: ```suggestion
This means that any user who wants to use sequences
```

> the user must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) 

Author: ```suggestion
must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) 
```
